go.mod

```diff
@@ -3,10 +3,6 @@ module github.com/cli-playground/devfile-parser
 go 1.14
 
 require (
-	github.com/docker/docker v1.13.1
```

Comment:
Please do `go mod tidy` to reflect these changes on `go.sum`

Reply:
done. WIll be adding this to the github actions going further.